src/components/atoms/SpecialServiceCard/SpecialServiceCard.module.scss

```diff
@@ -0,0 +1,25 @@
+@use "sass:map";
+@import "../../../foundations/sass/all";
+
+.service-card {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  gap: 15px;
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+
+  &__Title {
+    color: rgb(177, 6, 6);
+    @include center;
+    font-size: 50px;
+    margin-bottom: 25px;
+    font-weight: 900;
```

Comment:
Whenever possible, please utilize variables from the foundations folder. And It should 700 instead of 900 fw-bold

Reply:
got it